File: Java/LLStack.java
public class LLStack<item> {
    private class Node {
        item Item;
        Node next;
    }
    private Node first = null;

    public void push(item val){
        Node oldfirst = first;
        first = new Node();
        first.Item = val;
        first.next = oldfirst;
    }
    public boolean isEmpty() {
        return first == null;
    }
    public item pop(){
        item Item = first.Item;
        first = first.next;
        return Item;
    }
    public static void main(String[] args){
        LLStack<String> stack = new LLStack<String>();
        stack.push("World");
        stack.push("Hello ");
        System.out.print(stack.pop()+stack.pop());
    }
}

File: Python/LLStack.py
#Note: I am a C++ developer trying to learn Python, so my main focus is getting the code to work.
class Node:
    def __init__(self,item,next=None):
        self.item = item
        self.next = next
class Stack:
    def __init__(self):
        self.first = None
    def push(self,item):
        self.first = Node(item,self.first)
    def pop(self):
        item = self.first.item
        self.first = self.first.next
        return item
    def isEmpty(self):
        return self.first == None

#Test
X = Stack()
X.push("World")
X.push("Hello ")
print(X.pop() + X.pop())
